packages/gitbook/src/lib/api.ts

```diff
@@ -127,7 +127,15 @@ export function withAPI<T>(client: GitBookAPI, fn: () => Promise<T>): Promise<T>
 }
 
 export type PublishedContentWithCache =
-    | ((PublishedContentLookup | PublishedSiteContentLookup) & {
+    | ((
+          | PublishedContentLookup
+          | (PublishedSiteContentLookup & {
+                /**
+                 * ID of the default site space variant in the URL lookup.
+                 */
+                defaultSiteSpace: string;
```

Comment:
Do we need this type? Why is not coming from the api?

Reply:
We'll have to bump the API client which I don't want to do now as its already being done in the cleanup PR that also addresses typing issues. I'll add a `TODO` here anyway